contracts/src/PBHSignatureAggregator.sol

```diff
@@ -27,10 +28,14 @@ contract PBHSignatureAggregator is IAggregator {
     /// @notice Thrown when the length of the signature is invalid.
     error InvalidSignatureLength(uint256 expected, uint256 actual);
 
+    /// @notice Thrown when a zero address is passed as the PBHEntryPoint.
+    error ZeroAddress();
```

Comment:
Can we either update this to `AddressZero()` or update the `PBHEntryPointImplV1` to use `ZeroAddress()` for consistency?